plugins/lenovo-firmware/fu-plugin-lenovo-firmware.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2021 Mario Limonciello <mario.limonciello@amd.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_BETTER_THAN, "bios");
```

Comment:
What's this for?

Reply:
My original thought of looking what mode it's operating in, but it's really not needed.  I'll drop it.